lib/cpp/antaris_api_client.cc

```diff
@@ -43,6 +43,8 @@ unsigned int api_debug = 0;
 
 extern char g_SSL_ENABLE;
 
+extern char g_KEEPALIVE_ENABLE;
```

Comment:
I believe this will be set based on some flag (remote_app or enable_keep_alive) in client sdk config file

Reply:
yes. sdk_env.conf is setting this flag